go/controller/config/rbac/role.yaml

```diff
@@ -130,3 +133,9 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - deployments.plural.sh
+  resources:
+  - secrets
```

Comment:
fwiw this looks wrong?

Reply:
Ye, noticed that in the controller kube builder comment. It is fixed now.